wagtail/admin/action_menu.py

```diff
@@ -159,13 +172,19 @@ def __init__(self, request, **kwargs):
         for hook in hooks.get_hooks('construct_page_action_menu'):
             hook(self.menu_items, self.request, self.context)
 
+        self.default_item = self.menu_items.pop() if self.menu_items else SaveDraftMenuItem(order=50)
+
+
     def render_html(self):
+        print(self.template)
```

Comment:
Please remove this

Reply:
Removed on https://github.com/gasman/wagtail/commits/rjpruitt16-SaveDraft